src/utils/log.rs

```diff
@@ -6,11 +6,14 @@
 //! To log using this system please utilize the [`wlr_log!`](../../macro.wlr_log.html) macro.
 
 use libc::c_char;
-use wlroots_sys::{wlr_log_importance, __va_list_tag, wlr_log_init};
+use log::{info, max_level, set_boxed_logger, set_max_level, Level, LevelFilter, Metadata, Record};
```

Comment:
Please import just `log` and use the functions and values with the full crate name (e.g. `log::info`, `log::Level`). This module also defines logging for wlroots-rs and it's otherwise very confusing.